### PR TITLE
Grammar improvement after signup.

### PR DESCRIPTION
**Pull request by NekoJonez**
Like I said in the trac ticket, I think the sentence flows a bit better now. I also adjusted the translator comment for make it even more clear.

Trac ticket: (https://core.trac.wordpress.org/ticket/58115)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-signup.php [hunk 1]
```diff
@@ -704,8 +704,8 @@ function confirm_user_signup( $user_name, $user_email ) {
 	<p><?php _e( 'But, before you can start using your new username, <strong>you must activate it</strong>.' ); ?></p>
 	<p>
 	<?php
-	/* translators: %s: Email address. */
-	printf( __( 'Check your inbox at %s and click the link given.' ), '<strong>' . $user_email . '</strong>' );
+	/* translators: %s: The user Email address. */
```
Comment (audrasjb) [inline src/wp-signup.php]: I don't think "Email" should stay capitalized :)

**Review by audrasjb — CHANGES_REQUESTED** [review]
Thanks for the PR! I added one small change request ;)

**Comment by NekoJonez** [comment]
@audrasjb Fixed! Thanks for that feedback :D

**Comment by NekoJonez** [comment]
@audrasjb I looked further and I found the exact same thing lower in the file... So, I fixed that as well.

**Review by audrasjb — APPROVED** [review]
Nice 👌

**Review by krupal-panchal — APPROVED** [review]

**Review by krupal-panchal — APPROVED** [review]

**Review by hz-tyfoon — APPROVED** [review]

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/55902

**Pull request closed without merging**